addons/stock_account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_stock_valuation_layer(env):
+    openupgrade.logged_query(
```

Comment:
Why not doing this in the pre-migration? In any case I prefer to check the column exists just in case we want to attempt to migrate after a fail on a previous try.

Reply:
In any fail try, changes are rolled-back, but don't do that, or you will lose end-migration scripts: #1879

Reply:
Well, I think he refers for testing purposes, you may want to continue from the failed point than start from zero.

Reply:
But then as said, the whole module changes are rolled-back. Other advice, always run half-migrated DBs with -u all.